src/cc/bpf_module.cc

```diff
@@ -101,6 +101,7 @@ class MyMemoryManager : public SectionMemoryManager {
 
 BPFModule::BPFModule(unsigned flags, TableStorage *ts)
     : flags_(flags),
+      rw_engine_enabled_(true),
```

Comment:
Can we do this via flags? like `rw_engine_enabled_(!(flags & BPF_DISABLE_RW_ENGINE))`? This way it would be easy to control the behavior through Python / C++ API (who already has flags as constructor argument of the `BPF` class) without need to change the interface, or make extra call to enable / disable is later?

Reply:
The flags is used for debugging purpose. We probably do not want to mix it with other non-debugging thing.

Also, we probably do not want to expose this rw_engine_disabled thing to python API, it is just confusing to python users and unnecessary exposing internal details.